src/styles/index.scss

```
@@ -1,3 +1,70 @@
+$cell-height: 100px;
+$cell-width: 100px;
```

You can create only one variable for this value
```suggestion
$cell-size: 100px;
```